imap_processing/codice/codice_l1a.py

```diff
@@ -719,6 +789,75 @@ def get_params(dataset: xr.Dataset) -> tuple[int, int, int, int]:
     return table_id, plan_id, plan_step, view_id
 
 
+def group_data(packets: xr.Dataset) -> list[bytes]:
+    """
+    Organize continuation packets into appropriate groups.
```

Comment:
Bummer, I didn't realize you also had continuation packets to deal with otherwise I would have pushed harder to get a space_packet_parser release out the door with that support.
https://github.com/lasp/space_packet_parser/pull/115

@vmartinez-cu I think is doing a grouping for HIT, but I'm not sure you can re-use much code because your decommed fields you're indexing into are different. But worth hearing what she has to say.